numbat/src/bytecode_interpreter.rs

```diff
@@ -113,6 +119,28 @@ impl BytecodeInterpreter {
                 };
                 self.vm.add_op(op);
             }
+            Expression::BinaryOperatorForDate(_span, operator, lhs, rhs, _type, result_type) => {
+                self.compile_expression(lhs)?;
+                self.compile_expression(rhs)?;
+
+                let op = if *result_type == DateOperationResult::Seconds {
```

Comment:
Could we figure this out from `_type`? This way, we might be able to get rid of `result_type`?

Reply:
Oh, good point, yes I think we can.  I will try this.  If so, that would be much cleaner